### PR TITLE
Fix some errors uncovered by flow

### PR DESCRIPTION
**Pull request by charlag**
I tried to properly type mithril's hyperscript to account for the fact
that you can pass lifecycle hooks as attributes but this does not work.

Neither is it possible to strictly define attributes to disallow
passing properties which are not there.

### CODE REVIEW

File: flow/mithril-stream.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// @flow
+
+// TODO: think if they should be covariant. They probably should not be because they're not readonly
```
Comment (johnbotris) [inline flow/mithril-stream.js:3]: TODO

File: src/settings/SettingsView.js [hunk 2]
```diff
@@ -312,7 +312,6 @@ export class SettingsView implements CurrentView {
 									label: 'importUsers_action',
 									click: () => showUserImportDialog(this._customDomains.getLoaded()),
 									icon: () => Icons.ContactImport,
-									color: ButtonColors.Nav
```
Comment (johnbotris) [inline src/settings/SettingsView.js:315]: why do you delete color?

Comment (charlag) [reply src/settings/SettingsView.js:315]: NavButton does not have color

File: flow/mithril-stream.js (hunk 1 above)
Comment (charlag) [reply flow/mithril-stream.js:3]: It was like that already

**Comment by charlag** [comment]
@johnbotris can you approve it pls?

**Review by johnbotris — APPROVED** [review]

**Pull request merged**